### PR TITLE
Update vanderbilt downtime

### PR DESCRIPTION
**Pull request by PerilousApricot**
Add a resource and a downtime for it

### CODE REVIEW

**Comment by PerilousApricot** [comment]
Hi OSG folks, I realized when trying to update the downtime that we don't have  resoruce yet for XRootD, so I added that one as well

File: topology/Vanderbilt University/Vanderbilt ACCRE/Vanderbilt.yaml [hunk 1]
```diff
@@ -477,4 +477,39 @@ Resources:
         Description: Generic squid service
     VOOwnership:
       CMS: 100
+Vanderbilt_Xrootd:
```
Comment (brianhlin) [inline topology/Vanderbilt University/Vanderbilt ACCRE/Vanderbilt.yaml:480]:
```suggestion
  Vanderbilt_Xrootd:
```

**Review by brianhlin — APPROVED** [review]
LGTM

File: topology/Vanderbilt University/Vanderbilt ACCRE/Vanderbilt.yaml [hunk 2]
```diff
@@ -477,7 +477,7 @@ Resources:
         Description: Generic squid service
     VOOwnership:
       CMS: 100
-Vanderbilt_Xrootd:
+  Vanderbilt_Xrootd:
```
Comment (PerilousApricot) [inline topology/Vanderbilt University/Vanderbilt ACCRE/Vanderbilt.yaml:480]: VIM, you have failed me again!!

File: topology/Vanderbilt University/Vanderbilt ACCRE/Vanderbilt.yaml [hunk 3]
```diff
@@ -477,4 +477,39 @@ Resources:
...
+          ID: 11107327543306be8c340cfa2281444a2d428318
+          Name: Andrew Malone Melo
+        Secondary:
+          ID: 22ae3f93d335f46cea9be2f7c9e0551f31df44bf
+          Name: Eric Appelt
+    Description: Xrootd Redirector
+    FQDN: xrootd-vanderbilt.sites.opensciencegrid.org
+    ID: 1402
+    Services:
+      Xrootd component:
```
Comment (brianhlin) [inline topology/Vanderbilt University/Vanderbilt ACCRE/Vanderbilt.yaml:510]:
```suggestion
      XRootD component:
```

You're not going to believe this but...

**Pull request merged**